server/db/knexfile.js

```diff
@@ -41,7 +41,20 @@ module.exports = {
       tableName: 'knex_migrations'
     }
   },
-
+  server_db_test: {

```

Comment:
We're not actually using this config. We should use the `test` config for our test environment.